src/main/java/net/smartcosmos/cluster/userdetails/domain/UserAuthenticationProperties.java

```diff
@@ -0,0 +1,21 @@
+package net.smartcosmos.cluster.userdetails.domain;
+
+import java.util.Arrays;
+import java.util.List;
+
+import lombok.Data;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Configuration properties for dummy user.
+ */
+@Data
+@ConfigurationProperties("smartcosmos.security.user")
+public class UserAuthenticationProperties {
+
+    private String name = "user";
+    private String password = "password";
+    private List<String> authorities = Arrays.asList("https://authorities.smartcosmos.net/things/create");
```

Comment:
The intent is to have the authorities read from the config.  This can be made to be all of the read authorities but it really doesn't matter since the authorities should be in the config.  Another thought is if a user does not place the authorities in the config then they will be very restricted and that should get some attention.